src/collections/_documentation/workflow/releases/index.md

```diff
@@ -334,3 +334,74 @@ You can also use our [API]({%- link _documentation/api/releases/post-release-dep
 ## Release Artifacts
 
 JavaScript and iOS projects can utilize release artifacts to unminify or symbolicate error stack traces. To learn more, please check out our [iOS]({%- link _documentation/clients/cocoa/index.md -%}#sentry-cocoa-debug-symbols) and [JavaScript]({%- link _documentation/platforms/javascript/sourcemaps.md -%}) docs.
+
```

Comment:
Can we move it to the separate page under the WORKFLOW AND INTEGRATIONS and have the Release Glossary on the same page as the Release metrics?